src/bootstrap/test.rs

```diff
@@ -1540,6 +1540,7 @@ note: if you're sure you want to do this, please open an issue as to why. In the
             }
         }
         cmd.env("RUSTC_BOOTSTRAP", "1");
+        cmd.env("RUSTC_FORCE_INCR_COMP_ARTIFACT_HEADER", "compiletest");
```

Comment:
Can we add a comment here with some rationale for setting this?

I might prefer this to be pushed down into compiletest, fwiw.

Reply:
Added the comment. I initially tried to add this to compiletest it self but didn't find a good place due to the `aux` mechanism. Then I found the other environment variables set in bootstrap so I added it here.